storage_service/templates/login.html

```diff
@@ -21,7 +22,7 @@
 </tr>
 </table>
 
-<input type="submit" value="login" class='btn btn-primary' />
+<input type="submit" value="{% trans "login" %}" class='btn btn-primary' />
```

Comment:
I figured so, since it's a user facing string. I guess it should be Log In (the verb) though - and similarly for the page title and log out button This would also match the main dashboard.